.github/workflows/dash-bmv2-ci.yml

```diff
@@ -54,8 +54,6 @@ jobs:
       run:  make docker-dash-p4c
     - name: Build P4 software switch (bmv2) and P4Info
       run:  DOCKER_FLAGS=$docker_fg_flags make p4
-    - name: Install DASH submodules recursively
-      run:  git submodule update --init --recursive
```

Comment:
This was `git submodule update --init` before my changes with the comment `SAI submodule update`. Not sure whether it's still needed..

Reply:
I think it isn't needed anymore.